README.md

```diff
@@ -87,23 +91,43 @@ publisher, _ := pubsub.NewPublisher(
 To configure using managed identity with Service Bus, refer to this [link](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-managed-service-identity).
 Note if you see errors for certain operation, you may have an RBAC issue. Refer to the built-in RBAC roles [here](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-managed-service-identity#built-in-rbac-roles-for-azure-service-bus).
 #### Using user assigned managed identity
+Using Identity ClientID
+```
+topicName := "topic"
+publisher, _ := pubsub.NewPublisher(
+    topicName,
+    pubsub.PublisherWithManagedIdentityClientID(serviceBusNamespaceName, managedIdentityClientID),
+)
+```
+Using Identity ResourceID
 ```
 topicName := "topic"
 publisher, _ := pubsub.NewPublisher(
     topicName,
-    pubsub.PublisherWithManagedIdentity(serviceBusNamespaceName, managedIdentityClientID),
+    pubsub.PublisherWithManagedIdentityResourceID(serviceBusNamespaceName, managedIdentityResourceID),
 )
 ```
 
 #### Using system assigned managed identity
+Using Identity ClientID
 ```
 // keep managed identity client ID empty to default using system assigned managed identity
 topicName := "topic"
 publisher, _ := pubsub.NewPublisher(
     topicName,
-    pubsub.PublisherWithManagedIdentity(serviceBusNamespaceName, ""),
+    pubsub.PublisherWithManagedIdentityClientID(serviceBusNamespaceName, identityClientID),
```

Comment:
should this be "" for system managed identity?

Reply:
yes. for clientID you specify it. if you pass empty, then it uses SystemIdentity.
the comment just says that if you keep it empty, that's what happens.